<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseInsertDialog</class>
 <widget class="QDialog" name="CaseInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  案件发生地点:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  案件编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  车牌号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  报案单位或个人:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  报案人电话:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  案件发生时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  案件原因:  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  损失概述:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  备注:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Commit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Reset">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>420</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Return">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_NumberEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CarNumberEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_UnitEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_TelEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_PlaceEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_OtherEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="Insert_Date">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-M-d</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出险登记</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_CauseEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_SummartEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>881</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>462</y>
     <width>881</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_14</zorder>
  <zorder>label_13</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>Insert_Commit</zorder>
  <zorder>Insert_Reset</zorder>
  <zorder>Insert_Return</zorder>
  <zorder>Insert_NumberEdit</zorder>
  <zorder>Insert_CarNumberEdit</zorder>
  <zorder>Insert_UnitEdit</zorder>
  <zorder>Insert_TelEdit</zorder>
  <zorder>Insert_PlaceEdit</zorder>
  <zorder>Insert_OtherEdit</zorder>
  <zorder>Insert_Date</zorder>
  <zorder>Insert_CauseEdit</zorder>
  <zorder>Insert_SummartEdit</zorder>
  <zorder>label_11</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
